### ISSUE TITLE
Primal enchant and efficiency not compatible on revolver in newest version 

### ISSUE THREAD

**Issue by jrpoints** [Modbder/ThaumicBases#39]
Tried the update out to see if the primal enchant and efficiency enchant worked together and they still don't. 

I also tried to make a new revolver to see if the fix wasn't retroactive but with primal and 2x efficiency it was still using 2 essentia per shot.

**Comment by Modbder** [comment]
What are levels of both primal and efficiency on the revolver?

**Comment by jrpoints** [comment]
My old pre fix revolver I had was 1x primal and 5x efficiency, and it was using 2 essentia per shot. I made a new one, in case the fix wasn't retroactive with 1x primal and 2x efficiency and it also was using 2 essentia per shot.

**Comment by codewarrior0** [comment]
I'm using ThaumicBases-2.1.189.32, and I have a revolver with both Primal I and Efficiency V.

I can get two shots from one essentia, but after every two shots, the gun goes "click" when used and just rotates the cylinders.

**Comment by Modbder** [comment]
Yes, that is what is supposed to be happening - the reload each 2 shots. I still can't track the bug down, or even prove it's existance...
